fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -583,6 +583,7 @@ dropMaterializedViewStatement
 
 alterMaterializedViewStatement
     : ALTER MATERIALIZED VIEW mvName=qualifiedName (refreshSchemeDesc | tableRenameClause | modifyTablePropertiesClause)
+    | ALTER MATERIALIZED VIEW mvName=qualifiedName statusDesc
```

Comment:
seems can add ACTIVE in (refreshSchemeDesc | tableRenameClause | modifyTablePropertiesClause | ACTIVE)

Reply:
statusDesc may have INACTIVE in the future, Keep it here for better expansion.
The reason for starting a separate line is that there are too many options for him, and it doesn’t look good together.